Products/CMFPlone/PloneTool.py

```diff
@@ -34,7 +34,7 @@
                         ManagePortal, ManageUsers, ModifyPortalContent, View
 from Products.CMFCore.interfaces import IDublinCore, IMutableDublinCore
 from Products.CMFCore.WorkflowCore import WorkflowException
-from Products.CMFDefault.DublinCore import DefaultDublinCoreImpl
+from .DublinCore import DefaultDublinCoreImpl
```

Comment:
Not anymore. ;-) I fixed those just now on master at f4322dc20bffb7c9bb5ed271f2e9bc4b5c40e560.
And to put my money/time where my mouth is, I fixed the new ones on this branch as well, at 8c0ad6cfa63256d58a4a02feabcedf0df60b39f1.